package.json

```
@@ -29,7 +29,7 @@
     "@electron-forge/cli": "^6.0.0-beta.63",
     "@electron-forge/maker-deb": "^6.0.0-beta.63",
     "@electron-forge/maker-dmg": "^6.0.0-beta.63",
-    "@electron-forge/maker-rpm": "^6.0.0-beta.63",
+    "@electron-forge/maker-rpm": "^6.0.0-beta.67",
```

```suggestion
    "@electron-forge/maker-rpm": "^6.0.0-beta.67",
```

Let's do this differently.